antsibull/data/docsite/role.rst.j2

```diff
@@ -0,0 +1,286 @@
+.. Document meta
+
+:orphan:
+
+{# If we can put together source and github repo, we could make the Edit me of github button work.
+   See meta.get("source") in Ansible's docs/docsite/_themes/sphinx_rtd_theme/breadcrumbs.html
+   for more information
+:source: @{ source }@
+-#}
+
+.. Anchors
+
+.. _ansible_collections.@{plugin_name}@_@{plugin_type}@:
+
+.. Anchors: aliases
+
+{# TODO: This assumes that aliases will be short names.
+   If they're FQCN, then we need to change this
+#}
+
+.. Title
+
+{% if entry_points.main and entry_points.main.short_description -%}
+{%   set title = plugin_name + ' -- ' + entry_points.main.short_description | rst_ify -%}
+{% else -%}
+{%   set title = plugin_name -%}
+{% endif -%}
+
+@{ title }@
+@{ '+' * title|length }@
+
+.. Collection note
+
+{% if collection == 'ansible.builtin' -%}
```

Comment:
Will we ever have roles in ansible core?

Reply:
No idea. I don't think this will happen short-term, and probably never long-term either. But if it does happen, the description will be less wrong than if this isn't specially handled :)